src/index.html

```diff
@@ -10,7 +10,40 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+
```

Comment:
- unnecessary comments should be removed
- there shouldn't be empty lines between parent and child elements